packages/api-plugin-fulfillment-method-shipping-flat-rate/src/util/isDestinationRestricted.js

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Needs tests

Reply:
Tests are present in `filterShippingMethods.test.js`
I added one for regex based addressPattern check